src/Microsoft.ML.LightGbm/LightGbmMulticlassTrainer.cs

```diff
@@ -58,10 +116,19 @@ internal LightGbmMulticlassClassificationTrainer(IHostEnvironment env, Options o
             int? numberOfLeaves = null,
             int? minimumExampleCountPerLeaf = null,
             double? learningRate = null,
-            int numberOfIterations = Trainers.LightGbm.Options.Defaults.NumberOfIterations)
-            : base(env, LoadNameValue, TrainerUtils.MakeU4ScalarColumn(labelColumnName), featureColumnName, exampleWeightColumnName, null, numberOfLeaves, minimumExampleCountPerLeaf, learningRate, numberOfIterations)
+            int numberOfIterations = Defaults.NumberOfIterations)
+            : this(env,
+                  new Options()
+                  {
+                    LabelColumnName = labelColumnName,
+                    FeatureColumnName = featureColumnName,
+                    ExampleWeightColumnName = exampleWeightColumnName,
```

Comment:
>ExampleWeightColumnName [](start = 20, length = 23)

WeightColumnName

Reply:
I think rename should happen in a different PR. 

---
In reply to: [266673053](https://github.com/dotnet/machinelearning/pull/2948#discussion_r266673053) [](ancestors = 266673053)

Reply:
+1

---
In reply to: [266674022](https://github.com/dotnet/machinelearning/pull/2948#discussion_r266674022) [](ancestors = 266674022,266673053)